### PR TITLE
Improved base map controls

### PR DESCRIPTION
**Pull request by OsiJr**
- Added proper custom tileserver support
- Improving copyright handling
- Added 'default' Euclideon servers
- Removed tile preview
- Shrunk icons back to 24px and changed font back to 16px

### CODE REVIEW

File: src/vcSettingsUI.cpp [hunk 1]
```diff
@@ -281,11 +281,53 @@ void vcSettingsUI_Show(vcState *pProgramState)
 
           if (pProgramState->settings.maptiles.mapEnabled)
           {
-            if (ImGui::Button(vcString::Get("settingsMapsTileServerButton"), ImVec2(-1, 0)))
-              vcModals_OpenModal(pProgramState, vcMT_TileServer);
-
             ImGui::Checkbox(vcString::Get("settingsMapsDEM"), &pProgramState->settings.maptiles.demEnabled);
 
+            //"settingsMapsAttribution"
```
Comment (OsiJr) [inline src/vcSettingsUI.cpp:286]: Why is this here?

File: src/vcModals.cpp [hunk 2]
```diff
@@ -46,140 +46,6 @@ void vcModals_DrawLoggedOut(vcState *pProgramState)
...
-  vcState *pProgramState = (vcState*)pProgramStatePtr;
-
-  char buf[256];
-  udSprintf(buf, "%s/0/0/0.%s", pProgramState->settings.maptiles.tileServerAddress, pProgramState->settings.maptiles.tileServerExtension);
-
-  int64_t imageSize;
-  void *pLocalData = nullptr;
-
-  if (udFile_Load(buf, &pLocalData, &imageSize) != udR_Success)
-    imageSize = -2;
```
Comment (int-Frank) [inline src/vcModals.cpp:60]: Is this an error code?

Comment (samsinsane) [reply src/vcModals.cpp:60]: This is deleted code?
